guides/v3.15.0/object-model/enumerables.md

```diff
@@ -20,7 +20,7 @@ where available.
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
         It is best practice to
-        <a href="https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/">
```

Comment:
It looks like the link points to `release` for `v3.8` to `v3.14` as well. Can you push an extra commit where you use a relative path instead? I haven't checked locally but the relative path may be,

```
../configuring-ember/disabling-prototype-extensions/
```

If not,

```
../../configuring-ember/disabling-prototype-extensions/
```